src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -171,16 +172,24 @@ public static function getOrderSystemStatus(): array
         $orderSystemStatus['last_synchronized_date'] = TaskModel::getLatestSuccessfulSynchronizedDateForType(TaskHandler::ORDERS_EXPORT);
 
         $orderSystemStatus['ids_with_failed_tasks'] = TaskModel::getFailedOrderIds();
-        $unsynchronizedOrders = wc_get_orders([
+
+        $orderStatuses = wc_get_order_statuses();
+        unset($orderStatuses['wc-cancelled']);
+        $orderQuery = new \WC_Order_Query([
             'meta_key' => OrderHandler::TO_BE_SYNCHRONIZED_META_KEY,
             'meta_value' => 'yes',
             'meta_compare' => '=',
             'orderby' => 'date_created',
             'order' => 'ASC',
         ]);
-
+        $orderQuery->set('status', array_keys($orderStatuses));
+        $unsynchronizedOrders = $orderQuery->get_orders();
```

Comment:
can we just get the order ids instead of full order?

Reply:
We need the orders' `date_created` as well, unfortunately there is no such thing as returning specific fields only. It's either ids only or full object.
![image](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/assets/18332309/3fdf89ca-dedc-4677-961a-852f18d81ee4)